lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +22,14 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    tokenized_list = []
+    for i in text.lower():
+        if i.isalpha():
+            tokenized_list += i
+    en_text = tokenized_list
```

Comment:
why do you create the `en_text` variable?

Reply:
done